cmd/license-header-checker/main.go

```diff
@@ -91,43 +120,54 @@ func processFiles(options config.Options) {
 	}
 
 	wg.Wait()
+
 	elapsed := time.Since(start)
-	printResults(files, processedFiles, elapsed.Milliseconds(), options)
+	printResults(files, processedFiles, skipped, licenseOk, licenseAdded, licenseReplaced, elapsed.Milliseconds(), options)
 }
 
-func processFile(path string, license string, options config.Options) bool {
+func processFile(path string, license string, options config.Options) (Operation, error) {
+	green := color.FgGreen.Render
```

Comment:
move colors to global module:

```
var (
    green = color.FgGreen.Render
    yellow = color.FgYellow.Render
    red = color.FgRed.Render
)
```
and if it is posible use semantic names, the value of a variable must reflect what is for... and not actual value